src/content/docs/telemetry-data-platform/get-started/introduction-new-relic-data-ingest-apis-sdks.mdx

```diff
@@ -17,32 +17,11 @@ There are many ways to get data into your New Relic account. Any New Relic user
 
-To browse all New Relic-built tools and solutions, see [New Relic integrations](https://newrelic.com/integrations).
+Some options for getting started: 
 
-## Agent APIs
+* Log into [one.newrelic.com](https://one.newrelic.com) and click **Add more data** to get some guidance on setting up New Relic solutions.
+* To browse our solutions, see [New Relic integrations](https://newrelic.com/integrations).
 
-Some of our monitoring solutions come with APIs and/or SDKs that allow you to customize the data reported and how it reports. For more information, see the relevant product:
+## Report custom data [#custom-data]
 
-* [APM agent APIs](/docs/apis/get-started/intro-apis/introduction-new-relic-apis#apm-api)
-* [Browser API](/docs/apis/get-started/intro-apis/introduction-new-relic-apis#browser-api)
-* [Mobile API](/docs/apis/get-started/intro-apis/introduction-new-relic-apis#mobile-api)
-* Infrastructure monitoring: the [Flex integration tool](/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration)
-
-## Telemetry SDKs [#telemetry-sdk]
-
-If our [more curated solutions](#integrations) don't work for you, our open source [Telemetry SDKs](/docs/data-ingest-apis/get-data-new-relic/new-relic-sdks/telemetry-sdks-send-custom-telemetry-data-new-relic) let you build your own solution. These SDKs are language wrappers for our data-ingest APIs (below) that let you send telemetry data to New Relic without requiring install of an agent.
-
-## APIs for sending metrics, traces, logs, and events [#data-ingest-apis]
-
-If our more curated solutions don't work for you, we also have data-ingest APIs:
-
-* [Trace API](/docs/telemetry-data-platform/get-data/apis/introduction-trace-api/)
-* [Event API](/docs/insights/insights-data-sources/custom-data/introduction-event-api)
-* [Metric API](/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/introduction-metric-api/)
-* [Log API](/docs/logs/new-relic-logs/log-api/introduction-log-api)
-
-To learn about the differences between these data types, see [Data types](/docs/using-new-relic/data/understand-data/new-relic-data-types).
-
-## New Relic One applications [#new-relic-one-apps]
-
-You can build entirely custom applications that reside in New Relic One and make use of any data you want. You can use existing open source New Relic One apps, or share your own with the open source community. For details, see [New Relic One applications](/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application).
+If you need to report data that our [agents and integrations](#integrations) don't provide, we give you the tools you need to bring in any type of data you need. To learn more, see [Intro to custom data](/docs/telemetry-data-platform/custom-data/intro-custom-data). 
```

Comment:
Mmm not sure about the second bit of the sentence. Could we make it more directive? "... don't provide, check out our tools to bring in any type of data you need."